fyle_accounting_mappings/models.py

```diff
@@ -195,14 +195,11 @@ def bulk_create_or_update_expense_attributes(
                     )
                 )
             else:
-                if update and (
-                        ('detail' in attribute and attribute['detail'] != primary_key_map[attribute['value']]['detail'])
-                        or
-                        ('active' in attribute and attribute['active'] != primary_key_map[attribute['value']]['active'])
-                    ):
+                if update:
                     attributes_to_be_updated.append(
                         ExpenseAttribute(
                             id=primary_key_map[attribute['value']]['id'],
+                            source_id=attribute['source_id'],
```

Comment:
source_id wasn't getting updated for old attributes
because of which Corporate Cards were not getting correctly mapped.